Update interfaces/webrtc.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/e0218fe/whatwg/idl/webrtc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/506822321

foolip commented:
This is effectively a revert, assigning to @henbos to make the change in the spec too. This PR should then be closed automatically.

foolip commented:
Blocked on https://github.com/w3c/webrtc-pc/pull/2099.

autofoolip committed:
Update interfaces/webrtc.idl

Source: https://github.com/tidoust/reffy-reports/blob/e0218fe/whatwg/idl/webrtc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/506822321
